react/index.native.js

```diff
@@ -1,3 +1,5 @@
+import 'es6-symbol/implement'; // Polyfill Symbol
```

Comment:
IIRC babel takes care of that. We are doing some specific things on native because babel wasn't polyfilling it for us.